libs/common/src/helpers/date.test.ts

```diff
@@ -91,4 +91,73 @@ describe('date helpers', () => {
       expect(result).toEqual(expected);
     });
   });
+
+  describe('addYear', () => {
+    it('should add one year provided date object', () => {
+      const year = 1;
+      const date = new Date('20-09-1989');
+      const oneYear = date.getFullYear() + year;
+
+      const result = addYear(year, date);
+
+      expect(result.getFullYear()).toBe(oneYear);
```

Comment:
I think this isn't a full unit test - it's just testing that the year of the result is the same as the expected year - we also want to check that the day/month is correct (or that the full date object is as expected).